apps/src/applab/applab.js

```diff
@@ -699,8 +703,8 @@ Applab.onMouseMoveDebugResizeBar = function (event) {
   //  we explicitly set bottom/height styles on the elements above, we need to
   //  do the same for these styles as well)
 
-  codeTextbox.style.minHeight = 'calc(50% - 21px)';
-  debugArea.style.maxHeight = 'calc(50% - 21px)';
+  codeTextbox.style.minHeight = 'calc(50% - 16px)';
```

Comment:
I don't feel super strongly.  That said, is 21px half of the height?  I can intuit it, but somebody arriving at the code in a couple years' time might be a bit confused.